### ISSUE TITLE
Should Chapters Raise Money Only for Themselves, or Collectively/Globally??

### ISSUE THREAD

**Issue by BethanyG** [pyladies/global-organizing#101]
Should we have chapters decide on fundraising individually? Or focus on a central collection?

**Comment by MmeMarieLouise** [comment]
I think a central collection is better

**Comment by econchick** [comment]
I think we should allow chapters to fundraise if they wish. But PyLadies should have a global/central fundraising effort that will then be able to support all chapters (like how the PSF supports those that request funds).

**Comment by MmeMarieLouise** [comment]
@econchick I agree that each chapter should have the freedom to fundraise, but how will we manage where the funds are stored? Maybe it could go to the pyladies account but then is earmarked only for that chapter. What do you think ?
